### PR TITLE
added eslint and gulp modules in package.json

### PR DESCRIPTION
**Pull request by gitdongguri**
This patch is for supporting eslint feature using gulp and travis.
Before using eslint and gulp modules, we have to define the dependency in package.json.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module.exports = {
+    "env": {
```
Comment (coconutperm) [inline .eslintrc.js]: Tab is 2 space. right?

File: scripts/run_gulp [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+source ./scripts/util
+
+function run_gulp() {
+  platform_name=$(get_platform_name)
+  
+  if $(is_windows); then
+    #TODO(gitdongguri): run gulp in windows
+    echo "ttt"
+  else
+  echo "eeee"
```
Comment (coconutperm) [inline scripts/run_gulp]: plz check indent

File: .eslintrc.js (hunk 1 above)
Comment (gitdongguri) [inline .eslintrc.js]: Okay! How can I modify patch that I've done pull request?..

Comment (romandev) [inline .eslintrc.js]: You can push additional changes into the same branch. :)

File: scripts/run_gulp (hunk 2 above)
Comment (gitdongguri) [inline scripts/run_gulp]: indent is right but useless log is the problem right?

**Pull request closed without merging**